modules/sdk-core/src/bitgo/wallet/iWallet.ts

```diff
@@ -34,11 +34,19 @@ export interface BuildConsolidationTransactionOptions extends PrebuildTransactio
   consolidateAddresses?: string[];
 }
 
+export interface BuildTokenEnablementOptions extends PrebuildTransactionOptions {
+  tokens: {
+    name: string;
+    address?: string;
+  }[];
```

Comment:
same lets document properties.

especially address as its not always clear. e.g. "Some coins like Solana require tokens be enabled on a specific address. If absent, we assume it is for the Wallet's root address"